src/script/components/Badges/Badges.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {CSSProperties} from 'react';
+
+import {ConversationProtocol} from '@wireapp/api-client/lib/conversation';
+
+import {
+  CertificateExpiredIcon,
+  CertificateRevoked,
+  ExpiresSoon,
+  MLSVerified,
+  ProteusVerified,
+} from '@wireapp/react-ui-kit';
+
+export enum MLSStatues {
+  VALID = 'valid',
+  NOT_DOWNLOADED = 'not_downloaded',
+  EXPIRED = 'expired',
+  EXPIRES_SOON = 'expires_soon',
+  NOT_ACTIVATED = 'not_activated',
+}
+
+interface BadgesProps {
+  conversationProtocol?: ConversationProtocol;
+  isMLSVerified?: boolean;
+  isProteusVerified?: boolean;
+  MLSStatus?: MLSStatues;
+  displayTitle?: boolean;
+}
+
+const badgeWrapper: CSSProperties = {
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const iconStyles: CSSProperties = {
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const title = (isMLSConversation = false): CSSProperties => ({
+  color: isMLSConversation ? 'var(--green-500)' : 'var(--blue-500)',
+  fontSize: '12px',
+  lineHeight: '14px',
+  marginRight: '4px',
+});
+
+export const Badges: React.FC<BadgesProps> = ({
```

Comment:
suggestion: I don't think `Badges` properly conveys the functionality here. How about naming it `VerificationBadge`?